Themes/default/languages/index.english.php

```diff
@@ -16,7 +16,7 @@
 $txt['lang_spelling'] = 'american';
 
 // Ensure you remember to use uppercase for character set strings.
-$txt['lang_character_set'] = 'ISO-8859-1';
+$txt['lang_character_set'] = 'UTF-8';
```

Comment:
Please keep this as is, our language editor already taking care of this part based on the output :)